core/store/src/trie/insert_delete.rs

```diff
@@ -1,17 +1,17 @@
+use super::mem::updating::{
+    GenericNodeOrIndex, GenericTrieUpdate, GenericUpdatedTrieNode, UpdatedTrieStorageNodeWithSize,
```

Comment:
Should we considering moving all files like resharding.rs and updating.rs from trie/mem to outside mem directory? They are not specific to memtries any more

Reply:
Yes, we should, this is roughly mentioned in follow-ups.